release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1145,6 +1145,13 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-15-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
```

Comment:
OCPBUGS-385 is for oc-mirror plugin. It doesn't relate to oc. I'm not sure if we should create a sub-section oc-mirror under here? But we cannot leave it directly under oc in my opinion.

Reply:
How about I add the updates in Image registry?